test/unit/legacy_wrappers/mongo_client.test.js

```diff
@@ -22,6 +24,89 @@ describe('legacy_wrappers/mongo_client.js', () => {
     await client.close();
   });
 
+  describe('calling the constructor with invalid types', () => {
+    it('passing boolean to the options', () => {
```

Comment:
can you say what it's supposed to do?

Reply:
Fixed the title and added a descriptive comment, the ignore behavior is something we may change for the driver so I want to leave the full context here if this breaks in a future version.